jsonschema/_validators.py

```diff
@@ -78,7 +79,11 @@ def items(validator, items, instance, schema):
             if len(instance) > len(schema['prefixItems']):
                 yield ValidationError("%r has more items than defined in prefixItems" % instance)
             else:
-                for error in validator.descend(instance, {'prefixItems': schema['prefixItems']}, path='items__prefixItems'):
+                for error in validator.descend(
+                        instance,
+                        {'prefixItems': schema['prefixItems']},
+                        path='items__prefixItems',
+                ):
```

Comment:
Belongs in another commit.

Reply:
Squashing the commits before merging is a better option here. @Julian what do you think?

Reply:
I won't squash commits as part of merging -- definitely will preserve the commit history exactly as it was. But if the point here is that this is unrelated functionality, definitely split it into its own PR, but I suspect it isn't given prefixItems is new in these drafts.